addons/mrp/migrations/10.0.2.0/post-migration.py

```diff
@@ -0,0 +1,228 @@
+# -*- coding: utf-8 -*-
+# © 2017 Paul Catinean <https://www.pledra.com>
+# Copyright 2017 Eficent <http://www.eficent.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openupgradelib import openupgrade
+
+
+def update_mrp_workcenter_times(cr):
+    """The time now in minutes instead of hours."""
+    cr.execute(
+        """
+        UPDATE mrp_workcenter
+        SET time_start = COALESCE(time_start, 0) * 60,
+            time_stop = COALESCE(time_stop, 0) * 60
+        """
+    )
+
+
+def map_mrp_production_state(cr):
+    """Get ids of old 'draft' MOs and map states.
+    :return: list of ids with all old draft MOs.
+    """
+    cr.execute(
+        """
+        SELECT id
+        FROM mrp_production
+        WHERE state = 'draft'
+        """
+    )
+    draft_ids = [r[0] for r in cr.fetchall()]
+    openupgrade.map_values(
+        cr,
+        openupgrade.get_legacy_name('state'), 'state',
+        [('draft', 'confirmed'),
+         ('in_production', 'progress'),
+         ('ready', 'planned'),
+         ], table='mrp_production', write='sql')
+    return draft_ids
+
+
+def create_moves_draft_mos(env, draft_ids):
+    env['mrp.production'].browse(draft_ids)._generate_moves()
+
+
+def map_mrp_workorder_state(cr):
+    legacy_state = openupgrade.get_legacy_name('state')
+    if openupgrade.column_exists(cr, 'mrp_workorder', legacy_state):
+        # if mrp_operations was installed
+        openupgrade.map_values(
+            cr,
+            legacy_state, 'state',
+            [('draft', 'pending'),
+             ('starworking', 'progress'),
+             ('pause', 'ready'),
+             ], table='mrp_workorder', write='sql')
+
+
+def update_stock_warehouse(env):
+    """Check for manufacturing warehouses and update them.
+    :returns: a map (dict) to populate MO's picking types.
+              keys: location_dest_id
+              values: corresponding manu_type_id
+    """
+    cr = env.cr
+    cr.execute(
+        """
+        SELECT location_dest_id 
+        FROM mrp_production 
+        GROUP BY location_dest_id;
+        """
+    )
+    ids = [r[0] for r in cr.fetchall()]
+    loc_ids = env['stock.location'].browse(ids)
+    wh_ids = []
+    loc_wh_map = dict()
+    for loc in loc_ids:
+        wh_id = loc.get_warehouse()
+        wh_ids.append(wh_id.id)
+        loc_wh_map[loc.id] = wh_id.id
+    # update manufacture_to_ressupply
+    cr.execute(
+        """
+        UPDATE stock_warehouse
+        SET manufacture_to_resupply = TRUE
+        WHERE id in %s
+        """ % (tuple(set(wh_ids)),)
```

Comment:
Replace `%` with `,`. It prevents 

```
  File "/home/odoo/buildout/ou100/parts/odoo/addons/mrp/migrations/10.0.2.0/post-migration.py", line 86, in update_stock_warehouse
    """ % (tuple(set(wh_ids)),)
  File "/home/odoo/buildout/ou100/parts/odoo/odoo/sql_db.py", line 154, in wrapper
    return f(self, *args, **kwargs)
  File "/home/odoo/buildout/ou100/parts/odoo/odoo/sql_db.py", line 231, in execute
    res = self._obj.execute(query, params)
ProgrammingError: syntax error at or near ")"
LINE 4:         WHERE id in (1,)
```
Probably best not to execute this code when there are no wh_ids (even if that is not very likely at this point)